src/logic/calculate.js

```diff
@@ -0,0 +1,11 @@
+import Operate from './operate';
+
+const Calculate = data => {
+  const { numOne, operation, numTwo } = data;
+
+  if (!numTwo) return numOne;
+
+  return Operate(numOne, numTwo, operation);
+};
+
+export default Calculate;
```

Comment:
this function should have 2 parameters
- data object (you already got that) and
- button name (you dont have this)
[[see requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone4/buttonAsArgument.png)]